internal/server/proto/server.proto

```diff
@@ -923,6 +924,22 @@ message Job {
 
   message ExecResult {}
 
+  // Used to start a platform's exec function within a runner. This is only used
+  // there are no long running instances for a deployment and can fail if the
+  // platform plugin does not provide an exec function.
```

Comment:
Do you mind adding to this plugin that callers interested in logs should use the `LogStream` (or w/e its called) API instead of this and this is only meant for internal usage.

Reply:
💯